editor/sidebar/post-status/index.js

```diff
@@ -40,15 +47,17 @@ class PostStatus extends Component {
 
 		return (
 			<PanelBody title={ __( 'Status & Visibility' ) }>
-				<div className="editor-post-status__row">
-					<label htmlFor={ pendingId }>{ __( 'Pending review' ) }</label>
-					<FormToggle
-						id={ pendingId }
-						checked={ status === 'pending' }
-						onChange={ onToggle }
-						showHint={ false }
-					/>
-				</div>
+				{ ! isPublished &&
```

Comment:
Good point, enabling it is not difficult to achieve, but yeah, it doesn't make much sense. I'm more in favour of having a "revert to draft" button here. cc @jasmussen